install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -1,66 +1,414 @@
-GlusterFS, for persistent storage of registry images and for application storage.
+To copy the `<qcow_url>`:
++
+.. Navigate to link:https://access.redhat.com/downloads/content/69/ver=/rhel---7/latest/x86_64/product-software[].
+.. In the *Product Software* tab, locate the *_Red Hat Enterprise Linux KVM Guest Image_*.
+.. Right-click the *Download Now* button and copy the URL.
++
+This link expires after some time. Save the link when you are ready to run the playbook.
+
+. Create the bastion virtual machine:
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+# ansible-playbook -i localhost create-bastion-machine-playbook.yaml -e @secure_vars.yaml --ask-vault-pass
+----
+
+. Log in to the Administration Portal and click menu:Compute[Virtual Machines] to verify that the *_rhel-bastion_* virtual machine was created successfully.
+
+== Installing {product-title} on Red Hat Virtualization
+
+You can install {product-title} using the bastion virtual machine:
+
+. Log in to *_rhel-bastion_*.
+
+. Create *_vars.yaml_* with the following content and update its parameters:
++
+[source,yml]
+----
+---
+# For detailed documentation of variables, see
+# openshift_ovirt: https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_ovirt#role-variables
+# openshift installation: https://github.com/openshift/openshift-ansible/tree/master/inventory
+engine_url: https://<Manager_VQDN>/ovirt-engine/api
+engine_user: admin@internal
+engine_password: "{{ engine_password }}"
+engine_insecure: false
+engine_cafile: /etc/pki/ovirt-engine/ca.pem
+oreg_user: "{{ oreg_auth_user }}"
+oreg_pass: "{{ oreg_auth_password }}"
+
+openshift_ovirt_vm_manifest:
+  - name: 'master'
+    count: 1
+    profile: 'master_vm'
+  - name: 'compute'
+    count: 0
+    profile: 'node_vm'
+  - name: 'lb'
+    count: 0
+    profile: 'node_vm'
+
+# Set 'openshift_ovirt_all_in_one' to 'false' to install the master and node VMs separately.
+openshift_ovirt_all_in_one: true
+openshift_ovirt_cluster: Default
+openshift_ovirt_data_store: data
+openshift_ovirt_ssh_key: "{{ lookup('file', '/root/.ssh/id_rsa_ssh_ocp_admin.pub') }}"
+
+public_hosted_zone:
+# Uncomment to disable install-time checks, for smaller scale installations
+#openshift_disable_check: memory_availability,disk_availability,docker_image_availability
+
+qcow_url: <1>
+image_path: /var/tmp
+template_name: rhelguest7
+template_cluster: "{{ openshift_ovirt_cluster }}"
+template_memory: 4GiB
+template_cpu: 1
+template_disk_storage: "{{ openshift_ovirt_data_store }}"
+template_disk_size: 100GiB
+template_nics:
+  - name: nic1
+    profile_name: ovirtmgmt
+    interface: virtio
+
+debug_vm_create: false
+wait_for_ip: true
+vm_infra_wait_for_ip_retries: 30
+vm_infra_wait_for_ip_delay: 20
+
+node_item: &node_item
+  cluster: "{{ openshift_ovirt_cluster }}"
+  template: "{{ template_name }}"
+  memory: "{{ vm_memory | default('8GiB') }}"
+  cores: "{{ vm_cores | default(2) }}"
```

Comment:
@rgolangh This kind of looks like we expect the user to provide their own values for `vm_memory` and `vm_cores`. However, we never prompt them to do so. I know this works even without user input. But since we never ask the user to provide values for those two variables, we might as well say:
```
memory: '8GiB'
cores: 2
```
and it would have literally the same function.

Reply:
+1